examples/native/native_test.c

```diff
@@ -1195,17 +1187,19 @@ int TPM2_Native_Test(void* userCtx)
     session[0].auth.size = 0;
     XMEMSET(session[0].auth.buffer, 0, sizeof(session[0].auth.buffer));
 
-#ifdef WOLFTPM_ST33
-    /* Enable TPM2_EncryptDecrypt2 command */
-    XMEMSET(&cmdIn.setCmdSet, 0, sizeof(cmdIn.setCmdSet));
-    cmdIn.setCmdSet.authHandle = TPM_RH_PLATFORM;
-    cmdIn.setCmdSet.commandCode = TPM_CC_EncryptDecrypt2;
-    cmdIn.setCmdSet.enableFlag = 1;
-    rc = TPM2_SetCommandSet(&cmdIn.setCmdSet);
-    if (rc != TPM_RC_SUCCESS) {
-        printf("TPM2_SetCommandSet failed 0x%x: %s\n", rc,
-            TPM2_GetRCString(rc));
-        goto exit;
+#if defined(WOLFTPM_ST33) || defined(WOLFTPM_AUTODETECT)
+    if (TPM2_GetVendorID() == TPM_VENDOR_STM) {
```

Comment:
EncryptDecrypt2 native test is working only with ST33?

Reply:
This is a vendor specific command provided by ST for enabling the TPM2_EncryptDecrypt2 command...